README.md

```diff
@@ -149,6 +149,22 @@ If both these parameters are present, the node will also start service SSL traff
       "Config": { // configuration passed to the binary inside the container
         "api": "v1"
       }
+    },
+    "custom-service": {
+      "InternalPort": 8080,
+      "ExternalPort": 2000,
+      "NeedsKeys": false, // should pass the keys as a file; **never** set it to true, default false (optional)
```

Comment:
Needs is less descriptive imho, while keys is not specific enough - it's not public keys we are talking about...

I'd say pkAccess or secretAccess. 

BTW - I thought we make the signer a privileged service name - so this is inferred and enforced by the service name...